data-raw/gene_expressions.R

```diff
@@ -53,7 +54,7 @@ tpm <- tpm_5xfad %>%
 
 # 5XFAD ----
 
-#synGet("syn22103212", downloadLocation = here::here("data-raw"))
+# synGet("syn22103212", downloadLocation = here::here("data-raw", "gene_expressions", "5xfad"))
```

Comment:
`synGet()` has a `version` argument -- since these files have already changed once, can we track which version you're using here?